### PR TITLE
Added RPM packages to website

### PR DESCRIPTION
**Pull request by alexey-milovidov**
For changelog. Remove if this is non-significant change.

Category (leave one):
- Build/Testing/Packaging Improvement

Short description (up to few sentences):
Added RPM packages to website.

### CODE REVIEW

**Comment by alexey-milovidov** [comment]
TODO: add to documentation (after we fix all the remaining issues: #5815).

File: website/index.html [hunk 1]
```diff
@@ -483,7 +491,7 @@ <h2>Like ClickHouse?</h2>
    target="_blank"
 ><div id="github">Fork me on GitHub</div></a>
 
-<script type="text/javascript" src="//yastatic.net/jquery/3.1.1/jquery.min.js"></script>
+<script type="text/javascript" src="https://yastatic.net/jquery/3.1.1/jquery.min.js"></script>
```
Comment (blinkov) [inline website/index.html]: if url scheme is omitted it's taken from current page url, so this change doesn't do much except adding 6 bytes to content

File: website/index.html [hunk 2]
```diff
@@ -510,6 +518,30 @@ <h2>Like ClickHouse?</h2>
         if (hostParts.length > 2 && hostParts[0] != 'test') {
             window.location.host = hostParts[0] + '.' + hostParts[1];
         }
+
+        var available_distributives = ['deb', 'rpm'];
```
Comment (blinkov) [inline website/index.html]: it's more like package formats rather than distributives
also maybe let's also add tar.gz?

File: website/index.html [hunk 3]
```diff
@@ -399,10 +399,10 @@ <h2 id="quick-start">Quick Start</h2>
 
     <p>System requirements: Linux, x86_64 with SSE 4.2.</p>
 
-    <p>Install packages for Ubuntu/Debian:</p>
+    <p>Install packages for <span class="distributive_selected" id="repo_deb">Ubuntu/Debian</span>, <span class="distributive_not_selected" id="repo_rpm">RedHat/CentOS</span>:</p>
```
Comment (blinkov) [inline website/index.html]: * I'd swap CentOS and RedHat 
* comma could be replaced with "or"

**Review by blinkov — CHANGES_REQUESTED** [review]

**Comment by blinkov** [comment]
> TODO: add to documentation (after we fix all the remaining issues: #5815).

there's already some documentation on this, we could adapt it without waiting for anything
cc @BayoNet

File: website/index.html [hunk 4]
```diff
@@ -413,6 +413,14 @@ <h2 id="quick-start">Quick Start</h2>
 
 sudo service clickhouse-server start
 clickhouse-client
+</pre>
+<pre id="instruction_rpm" style="display: none;">
+sudo yum install yum-utils
+sudo yum-config-manager --add-repo https://repo.yandex.ru/clickhouse/rpm/stable/x86_64
+sudo yum install clickhouse-server clickhouse-client
+
```
Comment (alesapin) [inline website/index.html]: You missed pub key import:
`$ rpm --import https://repo.yandex.ru/clickhouse/CLICKHOUSE-KEY.GPG`

File: website/index.html (hunk 1 above)
Comment (alexey-milovidov) [reply website/index.html]: It's required to opening from `file://` to work.

File: website/index.html (hunk 2 above)
Comment (alexey-milovidov) [reply website/index.html]: .tar.gz are not ready.

**Pull request merged**